tensorflow/python/keras/benchmarks/keras_examples_benchmarks/mnist_irnn_benchmark_test.py

```diff
@@ -112,6 +112,26 @@ def benchmark_irnn_mnist_bs_1024(self):
     self.report_benchmark(
         iters=run_iters, wall_time=wall_time, metrics=metrics, extras=extras)
 
+  def benchmark_irnn_mnist_bs_1024_gpu_1(self):
```

Comment:
Any reason to use `gpu_1` here?

Reply:
I would like to provide a different GPU benchmark in each model, not only gpu_2 in here. And the results should be fine.

Reply:
Yeah, with that said, could you try 3 or 4 GPUs? As 1 gpu is not really distributed training, here the goal is to measure dist-strat.

Reply:
Sounds great, I will change these two to use 3 or 4 gpus.